x/evm/types/expected_keepers.go

```diff
@@ -22,13 +23,15 @@ import (
 type BaseKeeper interface {
 	Logger(ctx sdk.Context) log.Logger
 
+	HasChain(ctx sdk.Context, chain string) bool
 	ForChain(chain string) ChainKeeper
 
 	SetPendingChain(ctx sdk.Context, chain nexus.Chain, p Params)
 	GetPendingChain(ctx sdk.Context, chain string) (PendingChain, bool)
 	DeletePendingChain(ctx sdk.Context, chain string)
 	InitGenesis(ctx sdk.Context, state GenesisState)
 	ExportGenesis(ctx sdk.Context) GenesisState
+	BurnerInfo(c context.Context, req *BurnerInfoRequest) (*BurnerInfoResponse, error)
```

Comment:
Is it necessary to add it here?

Reply:
I don't think so, I just decided to had it anyway since the file had to be updated regardless.

Reply:
Yes, this function over at module.go requires this function to be added to the interface:

```
// RegisterServices registers a GRPC query service to respond to the
// module-specific GRPC queries.
func (am AppModule) RegisterServices(cfg module.Configurator) {
	types.RegisterQueryServiceServer(cfg.QueryServer(), am.keeper)
}
```

Reply:
I think `module.go` should have a reference of the actual type instead of the interface. Interfaces are useful for unit tests and mocking, and here I can't imagine anywhere we would want to call `BurnerInfo` ourselves and mock it.

Reply:
refactored module so that `module.go` has the actual type